Sources/TokamakCore/MountedViews/MountedApp.swift

```diff
@@ -34,49 +34,43 @@ final class MountedApp<R: Renderer>: MountedCompositeElement<R> {
     mountedChildren.forEach { $0.unmount(with: reconciler) }
   }
 
-  private func mountChild<S: Scene>(_ childBody: S) -> MountedElement<R> {
```

Comment:
In practice `S` is always `_AnyScene` here as everything is type-erased by this point, so no need for it to be generic.